### ISSUE TITLE
Add term vertebra plana

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10441]
**Preferred term label:**
Vertebra plana

**Synonyms**
Pancake vertebra, silver dollar vertebra

**Definition (free text, please give PubMed ID)**
The term given when a vertebral body has lost almost its entire height anteriorly and posteriorly, representing a very advanced compression fracture. PMID: 12432108, Skorupka K, Knipe H, Sharma R, et al. Vertebra plana. Reference article, Radiopaedia.org (Accessed on 05 Apr 2024) https://doi.org/10.53347/rID-7428

![image](https://github.com/obophenotype/human-phenotype-ontology/assets/160679899/255116e4-386f-43ff-ab04-5d167bdb84fa)
Source: Lustosa L, Vertebra plana. Case study, Radiopaedia.org (Accessed on 05 Apr 2024) https://doi.org/10.53347/rID-155169

**Parent term (use hpo.jax.org/app)**
Vertebral compression fracture HP:0002953

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
PMID: 37189540
Osteogenesis imperfecta (omim166200)
Osteoporosis (omim166710)
Langerhans cell histiocytosis (omim604856)
Bruck syndrome 2 (OMIM609220) PMID15523624

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@azankl
Adding new term: 
- ID: HP:6001053
- label: Vertebra plana
- parents: [Vertebral compression fracture (HP:0002953)](https://hpo.jax.org/app/browse/term/HP:0002953)
- pmid: PMID:12432108
- def: he term given when a vertebral body has lost almost its entire height anteriorly and posteriorly, representing a very advanced compression fracture.
- comment: 
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10441
- synonyms: Pancake vertebra(NONE)|Silver dollar vertebra(NONE)
